crates/pixi_manifest/src/pypi/pypi_options.rs

```diff
@@ -8,6 +8,37 @@ use serde::{Serialize, Serializer, ser::SerializeSeq};
 use thiserror::Error;
 use url::Url;
 
+/// The strategy to use when considering pre-release versions during dependency
+/// resolution.
+#[derive(
+    Default,
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    Serialize,
+    strum::Display,
+    strum::EnumString,
+    strum::VariantNames,
+)]
+#[strum(serialize_all = "kebab-case")]
+#[serde(rename_all = "kebab-case")]
+pub enum PrereleaseMode {
```

Comment:
could we import this type from rattler-lock? or we declare a new one to create macros over it?